src/analysis/sensitivity.jl

```diff
@@ -1,9 +1,17 @@
 module sensitivity
 
-using Logging
-using DataFrames, NamedDims, AxisKeys
-using Distributions, HypothesisTests, Bootstrap, StaticArrays, FLoops
-
+using
```

Comment:
Blue Style Guide requires one using per line in alphabetical order (https://github.com/invenia/BlueStyle#module-imports)

Reply:
I know this, but I think the function import by theme suggestion is a better fit here ("Avoid splitting a single export over multiple lines; either define one export per line, or group them by theme.").

That said, I did a poor job of grouping them by theme.

Reply:
1. Options 1 and 2 below use the same space, so why not the second?

```
# 1
using A
      B
      
# 2
using A
using B
```

2. You can have blocks of `using`'s grouped by theme but inside each block group `using`'s in alphabetical order.

Reply:
Yes, doing that now! 🤣